tests/standalone/test-pthread/create_join.c

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
This file is copied from https://web.dev/articles/wasm-threads, had better declare the original link

Reply:
OK. Modified the header.